src/Microsoft.SqlTools.ServiceLayer/ShowPlan/Contracts/ExecutionPlanGraph.cs

```diff
@@ -0,0 +1,103 @@
+using System.Collections.Generic;
```

Comment:
copyright

Reply:
do need to push more changes to resolve this?